cypress/integration/admin/fileManager/uploadImage.spec.js

```diff
@@ -20,18 +68,288 @@ context("File Manager View - CRUD", () => {
+        cy.findByText("File upload complete.").should("exist");
+
+        // Open file details
+        cy.findByTestId("fm-list-wrapper").within(() => {
+            cy.findAllByTestId("fm-list-wrapper-file")
+                .first()
+                .within(() => {
+                    cy.findByTestId("fm-file-wrapper-file-info-icon").click({ force: true });
+                });
+        });
+
+        cy.get("span").contains("File details").should("be.visible");
+
+        // Edit file
+        cy.get("span").contains("Add tags...").click();
+        cy.get('input[placeholder="homepage asset"]').type("rv");
+        cy.get("ul > li > span").contains("rv").click();
+        cy.get('input[placeholder="homepage asset"]').type("rv");
+        cy.get("ul > li > span").contains("No results.").should("be.visible");
+    });
+
+    it("should test adding 5 tags per file", () => {
+        const fileDetails = [
+            {
+                fileName: "pngPicture.PNG",
+                fileType: "image/png",
+                tagValues: [
+                    "webiny1_img1",
+                    "webiny2_img1",
+                    "webiny3_img1",
+                    "webiny4_img1",
+                    "webiny5_img1"
+                ]
+            },
+            {
+                fileName: "sample_2.jpeg",
+                fileType: "image/jpeg",
+                tagValues: [
+                    "webiny1_img2",
+                    "webiny2_img2",
+                    "webiny3_img2",
+                    "webiny4_img2",
+                    "webiny5_img2"
+                ]
+            },
+            {
+                fileName: "sample.jpeg",
+                fileType: "image/jpeg",
+                tagValues: [
+                    "webiny1_img3",
+                    "webiny2_img3",
+                    "webiny3_img3",
+                    "webiny4_img3",
+                    "webiny5_img3"
+                ]
+            }
+        ];
+
+        cy.fmListFiles({}).then(files => {
+            for (let i = 0; i < files.length; i++) {
+                deleteFile();
+            }
+        });
+
+        // Upload 3 files and add 5 tags per file.
+        fileDetails.forEach(fileDetail => {
+            cy.findByTestId("fm-list-wrapper").dropFile(fileDetail.fileName, fileDetail.fileType);
+            cy.findByText("File upload complete.").should("exist");
+
+            cy.findByTestId("fm-list-wrapper").within(() => {
+                cy.findAllByTestId("fm-list-wrapper-file")
+                    .first()
+                    .within(() => {
+                        cy.findByTestId("fm-file-wrapper-file-info-icon").click({ force: true });
+                    });
+            });
+
+            cy.get("span").contains("File details").should("be.visible");
+
+            cy.get("span").contains("Add tags...").click();
+
+            cy.get('input[placeholder="homepage asset"]').type(fileDetail.tagValues[0]);
```

Comment:
Can we use testId here? I'm afraid if we change the placeholder, these tests will fail.